topic_folders/policies/immediate-response-checklist.md

```diff
@@ -0,0 +1,78 @@
+- [ ] Post pull requests (PRs), issues, or comments to the Instructor Training GitHub repository. Write or administrative access to repositories will be removed for the duration of the suspension. 
+- [ ] Post to the Trainers Slack channel or mailing lists. The suspended member will be removed from these lists for the duration of their suspension. 
+- [ ] Participate in review of applications for open instructor training or Trainer training. 
+Participate as a learner in instructor training.
+
+### Trainer Training
+The suspended member will not be allowed to participate in Trainer Training (if they are currently enrolled) and will not be accepted for Trainer Training (if they apply during the suspension period). Additionally, they will not be allowed to shadow a current Carpentries Trainer. They may apply or reapply for future rounds of Trainer Training. 
+
+### Member Organisation and Local Activities
+The suspended member shall not be involved in officially branded Carpentries workshops or activities at the member or local site for the duration of the suspension. We expect the suspended member to assert that they will be able to carry out this suspension accordingly and to notify the CoC committee if this suspension will interfere with their expected employment duties. 
+
+If the suspension does affect employment duties or the suspended member is the key contact for a member site, the member will work with The Carpentries Executive Director to determine a course of action.
+
+### Lesson Development and Maintenance
+The suspended member will not be allowed to post PRs, issues, or comments to any repository in the carpentries, including but not limited to datacarpentry, swcarpentry, library carpentry, data-lessons, and carpentrieslab organizations. Administrative or write privileges to repositories in these organizations will be removed for the duration of the suspension.
+
+If the suspended member is a Maintainer, their name will be removed from the lessons page and they will not appear on the Maintainers page for the duration of their suspension. 
+
+### Executive Council
+If the suspended member is a member of the Executive Council, they will not participate in Executive Council responsibilities and will be removed from the Executive Council repositories and list of Executive Council members on the website for the duration of their suspension. The Executive Council will review the CoC incident to determine if a suspension will affect their long term position on the Executive Council.
+
+### Committees, Task Forces, and Other Interactions
+Involvement in all committees, communities, and task forces (African Task Force, Carpentries en LatinoAmérica, etc.) will be suspended. 
+
+- [ ] The suspended member's ability to sign in to AMY to manage their own profile or other tasks will be revoked for the duration of their suspension.
```

Comment:
@elliewix @karinlag the reason why their ability to sign-in to AMY to manage their profile will be revoked is because:

> They could click "yes" to publishing their profile which would automatically put them on our pages for instructors, trainers, etc.

Per @maneesha (thanks Maneesha!)